src/main/resources/application.yaml

```diff
@@ -124,3 +124,13 @@ darts-gateway:
     external-service-basic-authorisation-whitelist: ${EXTERNAL_SERVICE_BASIC_AUTHORISATION_WHITELIST:#{null}}
     jwks-cache-refresh_period: ${JWKS_REFRESH_PERIOD:30m}
     jwks-cache-lifetime_period:  ${JWKS_LIFETIME_PERIOD:60m}
+
+  # Default payload logging to be disabled for add document payloads
```

Comment:
Is this comment correct? Isn't it only the Daily List types for addDocument?

Reply:
yeah I can correct this